zeebe/exporters/elasticsearch-exporter/src/main/java/io/camunda/zeebe/exporter/ElasticsearchClient.java

```diff
@@ -163,11 +164,15 @@ public boolean shouldFlush() {
    * @return true if request was acknowledged
    */
   public boolean putIndexTemplate(final ValueType valueType) {
-    final String templateName = indexRouter.indexPrefixForValueType(valueType);
+    return putIndexTemplate(valueType, VersionUtil.getVersionLowerCase());
```

Comment:
Do we ever want to use this? With Broker version?

Reply:
It's only used for tests right now and no the default implementation is always the VersionUtil version